--- common/stringhelp.c
+++ common/stringhelp.c
@@ -1165,26 +1165,6 @@
 }
 
 
-/* Concatenate the string S1 with all the following strings up to a
-   NULL.  Returns a malloced buffer with the new string or NULL on a
-   malloc error or if too many arguments are given.  */
-char *
-strconcat (const char *s1, ...)
-{
-  va_list arg_ptr;
-  char *result;
-
-  if (!s1)
-    result = xtrystrdup ("");
-  else
-    {
-      va_start (arg_ptr, s1);
-      result = do_strconcat (s1, arg_ptr);
-      va_end (arg_ptr);
-    }
-  return result;
-}
-
 /* Same as strconcat but terminate the process with an error message
    if something goes wrong.  */
 char *
